File: playground/src/main/java/com/jiamian/translation/entity/dto/FilesApiDTO.java
package com.jiamian.translation.entity.dto;

import com.google.common.collect.Maps;
import lombok.Data;

import java.util.Map;

/**
 * @author DingGuangHui
 * @date 2023/3/28
 */
@Data
public class FilesApiDTO {

	private String name = "";
	private int id;
	private double sizeKB;
	private String type = "";
	private String format;
	private String pickleScanResult;
	private String pickleScanMessage;
	private String virusScanResult;
	private String scannedAt;
	private String downloadUrl;
	private boolean primary=true;

	private Map hashes = Maps.newHashMap();

}

File: web/src/main/java/com/jiamian/translation/entity/dto/api/FilesApiDTO.kt
package com.jiamian.translation.entity.dto.api

import com.google.common.collect.Maps

/**
 * @author DingGuangHui
 * @date 2023/3/28
 */
class FilesApiDTO {
    var name: String? = ""
    var id: Long? = 0
    var sizeKB: Double? = 0.0
    var type: String? = ""
    var format: String? = null
    var pickleScanResult: String? = null
    var pickleScanMessage: String? = null
    var virusScanResult: String? = null
    var scannedAt: String? = null
    var downloadUrl: String? = null
    var primary: Boolean? = true
    var hashes: Map<*, *> = Maps.newHashMap<Any, Any>()
}